src/win/util.c

```diff
@@ -1380,6 +1380,12 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = uv_malloc(1);
```

Comment:
Seems to me this has to be released in client code, or is there some `deinit` function that I'm missing?